official/vision/beta/configs/__init__.py

```diff
@@ -20,3 +20,4 @@
 from official.vision.beta.configs import retinanet
 from official.vision.beta.configs import semantic_segmentation
 from official.vision.beta.configs import video_classification
+from official.vision.beta.projects.yt8m.configs import yt8m
```

Comment:
For this project, please move this import to yt8m_train.py. The motivation of this is to make the project folder only one-way deepened on the outside yt8m/ folder.